autoload/test/neovim_error_only.vim

```diff
@@ -1,11 +1,12 @@
 let s:buffer_suffix = ' # vim-test'
 
 function! test#neovim_error_only#strategy(cmd) abort
-  let l:options = {'on_exit': 's:on_exit_handler'}
+  let l:options = {'on_exit': function('s:on_exit_handler')}
 
   call s:close_buffer()
 
   belowright new | call termopen(a:cmd . s:buffer_suffix, l:options)
```

Comment:
You could probably add this to the pipe chain above, e.g., `belowright new | call termopen(a:cmd . s:buffer_suffix, l:options) | wincmd p`.